pylutron/__init__.py

```diff
@@ -103,8 +103,9 @@ def _do_login_locked(self):
     try:
       sock = self._telnet.get_socket()
       sock.setsockopt(socket.SOL_SOCKET, socket.SO_KEEPALIVE, 1)
-      # Send keepalive probes after 60 seconds of inactivity
-      sock.setsockopt(socket.IPPROTO_TCP, socket.TCP_KEEPIDLE, 60)
+      if hasattr(socket, 'TCP_KEEPIDLE'):
+        # Send keepalive probes after 60 seconds of inactivity
+        sock.setsockopt(socket.IPPROTO_TCP, socket.TCP_KEEPIDLE, 60)
```

Comment:
out of curiosity, which linux platforms don't have keepidle? It's been there since 2.4, how old is your kerne?!

Reply:
You know what - I might have just made a huge leap when reading the docs + seeing it not work on macOS:

> On Windows, TCP_KEEPIDLE, TCP_KEEPINTVL appear if run-time Windows supports.

I think I quickly skimmed and translated to "this only exists on Windows". It definitely throws AttributeError on macOS though.

Will update description accordingly.